src/Classical/linear_code.jl

```diff
@@ -663,7 +663,6 @@ end
 Return whether or not `v` is a codeword of `C`.
 """
 in(v::Union{CTMatrixTypes, Vector{Int}}, C::AbstractLinearCode) = iszero(syndrome(C, v))
-∈(v::Union{CTMatrixTypes, Vector{Int}}, C::AbstractLinearCode) = iszero(syndrome(C, v))
```

Comment:
the repetition of ```in``` causes a precompile error. Probably its there because of a merge error so I deleted one of them